src/components/accordion/accordion.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { Accordion, AccordionSection } from './accordion';
+import '@reach/accordion/styles.css';
+import { ILLUSTRATION_NAMES } from '../illustration/illustration-set';
+
+export const Regular = () => {
+  return (
+    <Accordion>
+      <AccordionSection
+        title="1. Deine Bewerbung"
+        illustration={'scientistB_007'}
+      >
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab dolorem
+        itaque nam, quisquam quos sed totam voluptas voluptatem voluptatibus.
+        Debitis dicta dolores enim esse fugit impedit odio sed sunt tenetur!
+      </AccordionSection>
+      <AccordionSection
+        title="2. Das Kennenlernen"
+        illustration={'astronaut_015'}
+      >
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab dolorem
+        itaque nam, quisquam quos sed totam voluptas voluptatem voluptatibus.
+        Debitis dicta dolores enim esse fugit impedit odio sed sunt tenetur!
+      </AccordionSection>
+      <AccordionSection title="3. Abschluss">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab dolorem
+        itaque nam, quisquam quos sed totam voluptas voluptatem voluptatibus.
+        Debitis dicta dolores enim esse fugit impedit odio sed sunt tenetur!
+        consectetur adipisicing elit. Ab dolorem itaque nam, quisquam quos sed
+        totam voluptas voluptatem voluptatibus. Debitis dicta dolores enim esse
+        fugit impedit odio sed sunt tenetur!
+      </AccordionSection>
+    </Accordion>
+  );
+};
+
+interface AccordionExampleProps {
+  defaultIndex: number;
+}
+
+export const AccordionContainerExample = (props: AccordionExampleProps) => {
+  return (
+    <Accordion defaultIndex={[0, 1]}>
+      <AccordionSection title="Section A">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab dolorem
+        itaque nam, quisquam quos sed totam voluptas voluptatem voluptatibus.
+        Debitis dicta dolores enim esse fugit impedit odio sed sunt tenetur!
+      </AccordionSection>
+      <AccordionSection title="Section B">
+        This section is opened by default as we defined the defaultIndex to be
+        1.
+      </AccordionSection>
+    </Accordion>
+  );
+};
+
+export const AccordionSectionExample = (args) => {
+  return (
+    <Accordion defaultIndex={[0, 1]}>
```

Comment:
`defaultIndex` is the internal reach ui name. You can pass in a number of an array of number to tell what is opened on the initial rendering. You can't control the panels during runtime. 

I thought about renaming this but I found no better words so I pick the same name as the internal one from reach ui.